psocake/gui.py

```diff
@@ -491,6 +567,20 @@ def mouseClicked(evt):
 
 def main():
     global ex
+    global eventDocument
+    eventDocument = {}
+    global hit_dict
+    hit_dict = { QtCore.Qt.Key_4 : 0,   # No     hit (alias)
+                 QtCore.Qt.Key_0 : 0,   # No     hit
+                 QtCore.Qt.Key_1 : 1,   # Single hit
+                 QtCore.Qt.Key_2 : 2,   # Multi  hit
+                 QtCore.Qt.Key_3 : 3, } # Unknow hit
+    global hit_interpret_dict
+    hit_interpret_dict = { 0 : 'NO',
```

Comment:
Hard coding these labels is ok for SPI labeling. But what if another psocake user wants to label something else for machine learning? A generalizable interface would be nice.

Reply:
That's a good point I missed.  Hmm... there needs to be a better solution.  Maybe like a prompt for user to type?